hector_math/docs/conf.py

```diff
@@ -231,7 +231,7 @@
 # POPULATE LINKS TO OTHER VERSIONS
 html_context["versions"] = list()
 
-versions = [branch.name for branch in repo.branches]
+versions = [x.replace("/", "_") for x in [branch.name for branch in repo.branches]]
```

Comment:
I would recommend not making your own solution for stuff like that.
Identify in what way it needs to be escaped: Does it have to be part of a valid URL, or a valid file/folder name?
Find a library that escapes your string for that purpose and use that.
Otherwise, you are bound to find cases you've missed all the time.